web_src/js/features/repo-code.js

```diff
@@ -25,7 +25,9 @@ function getLineEls() {
 }
 
 function selectRange($linesEls, $selectionEndEl, $selectionStartEls) {
-  $linesEls.closest('tr').removeClass('active');
+  for (const el of $linesEls) {
+    el.closest('tr').classList.remove('active');
+  }
```

Comment:
Seems a bit unusual to iterate a jQuery collection with `for-of`, but if it works I can accept it.